### PR TITLE
RELATED: FET-148 translation metadata

### PR DESCRIPTION
**Pull request by yanlipnican**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1946/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1430/ : FAILURE in 5m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1948/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1432/ : SUCCESS in 1m 56s

File: src/components/core/base/IntlWrapper.tsx [hunk 1]
```diff
@@ -42,6 +42,15 @@ export const messagesMap = {
 
 export const DEFAULT_LOCALE = 'en-US';
 
+export function removeMetadata(translationsWithMetadata: any) {
```
Comment (tomassatinsky) [inline src/components/core/base/IntlWrapper.tsx]: please find just one place for that function and reuse it

Comment (tomassatinsky) [reply src/components/core/base/IntlWrapper.tsx]: or even add it to https://github.com/gooddata/gdc-js-utils/

Comment (tomassatinsky) [reply src/components/core/base/IntlWrapper.tsx]: so it can be re-user acroaa all the affected repos

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by martinmosko** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/823/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2026/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/454/ : FAILURE in 34s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/824/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2027/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/455/ : UNSTABLE in 19m 51s

**Comment by yanlipnican** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/825/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2028/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/456/ : SUCCESS in 16m 47s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/468/ : SUCCESS in 15s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1497/ : SUCCESS in 6m 56s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/827/ : SUCCESS in 17m 58s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/409/ : SUCCESS in 17m 45s

**Pull request merged**